### PR TITLE
Remove unnecessary case in uninstall script

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/contributing/).

---
**What does this PR aim to accomplish?:**
Does what it says.
Fixes https://github.com/pi-hole/pi-hole/issues/4691

### CODE REVIEW

**Review by PromoFaux — APPROVED** [review]

File: automated install/uninstall.sh [hunk 1]
```diff
@@ -14,7 +14,6 @@ while true; do
     read -rp "  ${QST} Are you sure you would like to remove ${COL_WHITE}Pi-hole${COL_NC}? [y/N] " yn
     case ${yn} in
```
Comment (dschaper) [inline automated install/uninstall.sh:15]: Need to take out the `n` option here as well.

Comment (yubiuser) [reply automated install/uninstall.sh:15]: Why? `yn` is only the name of the variable

Comment (PromoFaux) [reply automated install/uninstall.sh:15]: maybe `response` or `answer` instead? 🤷‍♂️

Comment (yubiuser) [reply automated install/uninstall.sh]: Done.

**Review by PromoFaux — APPROVED** [review]

**Pull request merged**